nearlib/dev.js

```diff
@@ -14,33 +14,33 @@ module.exports = {
     getConfig: async function() {
         return JSON.parse(decodeURIComponent(getCookie('fiddleConfig')));
     },
-    connect: async function(nodeUrl) {
-        const studioConfig = await this.getConfig();
-        const near = nearlib.Near.createDefaultConfig(nodeUrl || studioConfig.nodeUrl);
-        await this.getOrCreateDevUser(near);
-        return near;
-    },
     /**
-     * Create a connection which can perform operations on behalf of alice.near. This is an
-     * account that only exists on dev enviornments, so this should never be called on mainnet.
-     * @param {string} nodeUrl
+     * @param {String} nodeUrl
      */
-    connectWithAlice: async function(nodeUrl) {
-        return this.connectWithAccount(nodeUrl, aliceAccountName, aliceKey);
+    connect: async function(nodeUrl) {
+        const options = {};
+        options.node_url = nodeUrl;
+        return await this.connect(options);
```

Comment:
this looks like infinite recursion

Reply:
let's only have method that takes options, it's not hard to write `connect({nodeUrl: "...."})`

Reply:
``` 
near = await nearlib.dev.connect();
```

Don't fiddles use method without params? It should be able to still work without params when taking options.

Reply:
no they do not. in fact if me make the default localhost, fiddles that don't pass params will definitely NOT work

Reply:
well this PR needs to go in to unblock 5 other things, so I added a default value.

Reply:
`const nodeUrl = options.nodeUrl || this.getConfig().nodeUrl || 'http://localhost:3030';`

won't this use proper url for fiddle when no options are passed?